src/platform/nrf52/NRF52Bluetooth.cpp

```diff
@@ -295,6 +295,7 @@ bool NRF52Bluetooth::onPairingPasskey(uint16_t conn_handle, uint8_t const passke
 {
     LOG_INFO("BLE pairing process started with passkey %.3s %.3s\n", passkey, passkey + 3);
     powerFSM.trigger(EVENT_BLUETOOTH_PAIR);
+    #ifndef MESHTASTIC_EXCLUDE_SCREEN
```

Comment:
shouldn't this read `#if !MESHTASTIC_EXCLUDE_SCREEN`

Reply:
Good call.